modules/open_data_schema_dcat/dcatValidator.php

```diff
@@ -1,94 +0,0 @@
-<?php
```

Comment:
@kducharm I note for next time. It would be useful to isolate file name changes and other renaming or clean-up-type of tasks to their own commit. That would make it much easier to see the actual functional changes in the codebase.